dev/app-shell/elements/arc-app.html

```diff
@@ -566,6 +566,18 @@
     _onSuggest(e, suggestion) {
       this._setState({plan: suggestion.plan});
     }
+    _onMakeSuggestions(e) {
+      let search = e.detail.search;
+      if (search)
+        search = search.trim().toLowerCase();
+      if (!search)
+        search = null;
+
```

Comment:
At least up to now, there are now blank lines inside of functions in this codebase. This is all subjective, but I find it harder to read that way, and generally, if I want a blank line, it indicates to me I probably should factor into smaller bits. I'm not insisting on this, just noting it.

Reply:
done